subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -845,6 +847,28 @@ public void useJUnit(Action<? super JUnitOptions> testFrameworkConfigure) {
         useTestFramework(new JUnitTestFramework(this, (DefaultTestFilter) getFilter()), testFrameworkConfigure);
     }
 
+    /**
+     * Specifies that JUnit platform should be used to execute the tests. <p> To configure JUnit platform specific options, see {@link #useJUnitPlatform(Action)}.
```

Comment:
:x: I think we'll need to explain that JUnit platform is JUnit 5 here. Otherwise I fear that user do not know what it is exactly.

Reply:
Yes, I'll complete the documentation soon.